tests/test_modeling_bart.py

```diff
@@ -570,144 +570,82 @@ def test_cnn_summarization_same_as_fairseq(self):
         # TODO(SS): add test case that hits max_length
 
     def test_prepare_seq2seq_batch(self):
-        tokenizer = self.default_tokenizer
+        tokenizers = [self.default_tokenizer, self.default_tokenizer_fast]
```

Comment:
@sshleifer , used for loop and tested both tokenizers in same method

Reply:
nice.